### PR TITLE
backbone shufflenet module

### PR DESCRIPTION
**Pull request by 315386775**
Reproduced shufflenet V1 & V2 module. 

Reference ： https://github.com/megvii-model/ShuffleNet-Series

| Model | TOP1 | TOP5 | Note | 
| ----- | ----- | ----- | ----- | 
| ShuffleNetV1 1.0x  | 32.2 | 12.3 | pytorch official| 
| ShuffleNetV1 | 32.8 | 12.7 | gluoncv | 
| ShuffleNetV2 1.0x| 30.6 | 11.1 | pytorch official | 
| ShuffleNetV2  | 31.1 | 11.5 | gluoncv |

### CODE REVIEW

**Comment by 315386775** [comment]
Resubmit

File: README.md [hunk 1]
```diff
@@ -190,3 +190,18 @@ If you feel our code or models helps in your research, kindly cite our papers:
   year={2020}
 }
 ```
+
+# New backbone shufflenet module.
```
Comment (bryanyzhu) [inline README.md:194]: Please remove this training bash from README, we will add it later to docs page (https://cv.gluon.ai/model_zoo/classification.html). Thank you.

**Review by bryanyzhu — CHANGES_REQUESTED** [review]
Awesome PR. If you have trained weights, can you share the (1) model weights and (2) numbers on ImageNet with me? I will upload them to our model store. Thank you.

**Comment by bryanyzhu** [comment]
Syntax check has failed, please fix these issues to move on. Let me know if you need any help, thanks.

**Comment by 315386775** [comment]
@bryanyzhu thanks. this is my first time to contribute. i want to know :
a. i can share the trained model weights and train logs. where can i upload this files？
b. Syntax check has failed. maybe i need your help. i have pulled the new gluoncv version.

**Comment by bryanyzhu** [comment]
> @bryanyzhu thanks. this is my first time to contribute. i want to know : a. i can share the trained model weights and train logs. where can i upload this files？ b. i have fixed the error according to the action logs.

A. You can upload your bash/models/logs to any cloud service, like google drive, onedrive, dropbox, and let me know the link. I will then upload them to our own S3 server. 

B. Thank you for fixing the errors. It seems that there are still some warnings. If you click on the red check again, it says (1) several lines are too long (2) trailing white space (3) and bad white space. 

Regarding (1), you can simply add `# pylint: disable=line-too-long` to the beginning of file (sth like [here](https://github.com/dmlc/gluon-cv/blob/ab03ca04c588342be5cd659c3f96011c0146ac4f/gluoncv/utils/compress_json.py#L1)). This will ignore the warning. 

Regarding (2), you can just remove the extra white spaces in the end of that line (shufflenet.py:120:0: C0303: Trailing whitespace (trailing-whitespace))

Regarding (3), you can remove the bad white space after `print` (shufflenet.py:204:14: C0326: No space allowed before bracket print ('x:', x))

Hope this will pass the syntax test.

**Review by bryanyzhu — APPROVED** [review]

**Comment by 315386775** [comment]
@bryanyzhu i have upload the bash/models/logs to dropbox. Link:
[https://www.dropbox.com/sh/8j2lry46msdvzsf/AACzkZEInYMEaAmd3sCNTXIua?dl=0](url)

Hyperparameters can continue to be adjusted, and I will continue to follow up.
